lesson02/canDrive.js

```diff
@@ -0,0 +1,25 @@
+const firstName = "Oyun";
+const lastName = "Batnasan";
+const age = 29;
+const drivingAge = 16;
+const drivingTime = age - drivingAge;
+
+
+//check to see if I'm illegal to drive
+//if I'm prin Oyun Batnasan can drive and has driven for x years
+if (age >= drivingAge) {
+  //do codes
+  console.log(firstName + ' ' + lastName + ' ' + 'can drive and has driven for ' + drivingTime + 'years')
+} else
+
+  //if I'm not legal to drive console log down from 5
+
+  console.log('5');
```

Comment:
extra indentation here...plus all these could go away as you're handling the logs in the loop